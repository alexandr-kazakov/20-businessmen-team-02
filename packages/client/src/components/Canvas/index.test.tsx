import React from 'react'
import { render } from '@testing-library/react'

import CanvasComponent from '@/components/Canvas/index'

test('should render canvas component', () => {
  const {asFragment} = render(
    <CanvasComponent
      className=''
      setScores={jest.fn()}
      level='puzzleLevel'
      initStart={0}
      src=''
    />
  )

  expect(asFragment()).toMatchSnapshot()
})
