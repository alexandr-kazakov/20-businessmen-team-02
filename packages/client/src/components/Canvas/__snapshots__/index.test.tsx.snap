// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render canvas component 1`] = `
<DocumentFragment>
  <canvas
    class=""
    height="NaN"
    width="NaN"
  />
</DocumentFragment>
`;
